lib/rubocop/rspec/rspec_config.rb

```diff
@@ -89,10 +89,12 @@ def rspec_keywords_all_all
       end
 
       def rspec_pattern
+        return @rspec_pattern if @rspec_pattern
```

Comment:
Is this memoization a necessary part of this PR? It’s been discussed previously in #967, and as I recall it, memoizing on the Cop instance doesn’t make much of a speed difference.

Reply:
I just glimpsed that PR and thought that since I'm moving configuration-related methods from Cop to Config anyway I could add memoization there. `RSpecConfig` is included into `RuboCop::Config` in this PR, so it will be memoization on config level.

Anyway if you think that it's not quite needed to this particular PR, I can rebase without it, thankfully it's a standalone commit.